atgu/atgu/templates/resource.html

```diff
@@ -0,0 +1,62 @@
+{% extends "layout.html" %}
+{% block title %}{{ resource['title'] }}{% endblock %}
+{% block head %}
+  <link href="https://cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet" crossorigin="anonymous">
+{% endblock %}
+{% block breadcrumb_items %}
+  <li class="breadcrumb-item"><a href="{{ base_path }}">ATGU</a></li>
+  <li class="breadcrumb-item"><a href="{{ base_path }}">Resources</a></li>
+  <li class="breadcrumb-item active" aria-current="page">{{ resource['title'] }}</li>
+{% endblock %}
+{% block container_contents %}
+  <div class="row">
+    <div class="col">
+      <div style="float:right;">
+        <form action="{{ base_path }}/resources/{{ resource['id'] }}/edit" class="pr-3" style="display:inline;">
+          <button type="submit" class="btn btn-primary">Edit</button>
+        </form>
+
+        <form action="{{ base_path }}/resources/{{ resource['id'] }}/delete" method="post" style="display:inline;">
+          <input type="hidden" name="_csrf" value="{{ csrf_token }}">
```

Comment:
I think you're missing the description

Reply:
sorry. I think I put the comment in the wrong place. I think I meant here

```
		   <div class="mb-3">
     <div id="contents"></div>
   </div>
```

But further I tried to update the description on the edit page so I could double check here and it ignored my update (went through but if I refreshed the page the description was gone). This was from an old deployment so maybe it's working now.

Reply:
Ah, that was intentional.  I meant the description to be a short summary for the search page, but since the actual content is in the /resource page, I thought the summary was redundant there so I left it out.